packages/react-split/index.d.ts

```diff
@@ -19,6 +19,7 @@ export interface SplitProps {
     onDragStart?: Options["onDragStart"]
     onDragEnd?: Options["onDragEnd"]
     collapsed?: Number
+    className?: string
```

Comment:
This works only for `className` but there are many other attributes for `div` element such as `style`, `data-*`, etc. 
You can have `SplitProps` interface (line 4) extends `React.HTMLAttributes<HTMLDivElement>` instead to inherit all div element attributes.